### DISCUSSION TITLE
Custom element not found

### DISCUSSION THREAD

**Discussion by scienceguyroy** [dbuezas/lovelace-plotly-graph-card#472] [Q&A] [answered]
Hello, after installed the card through HACS, the card is not available for dashboards.  I also tried editing the YAML code directly, and I got an error message, "Custom element not found: plotly-graph."  I do see its URL in the dashboard resources page.  Not sure what I'm missing!  Any assistance would be appreciated!

**Comment by dbuezas** [comment]
Have you reloaded the frontend? (Reload the page)
You could also try restarting home assistant and checking again in hacks to ensure it installed correctly

**Reply by scienceguyroy** [reply, marked as answer]
Thank you for the reply!  I redownloaded and now, it's there.  Sorry for the trouble!

**Reply by dbuezas** [reply]
No problem! Good luck
